src/pages/Consultations/fields.tsx

```diff
@@ -0,0 +1,28 @@
+import ProCard from '@ant-design/pro-card';
+import { PageContainer } from '@ant-design/pro-layout';
+import React from 'react';
+import { FormattedMessage, history } from 'umi';
+
+import ModelFields from '@/components/ModelFields';
+
+const ConsultationFields: React.FC = () => {
+  return (
+    <PageContainer>
+      <ProCard
+        tabs={{
+          type: 'card',
+          activeKey: 'fields',
+          onChange: () => history.push('/other/consultations'),
```

Comment:
Here in onChange function you should get key argument and set in url:
onChange: (key) => history.push('/other/consultations/${key}'), 
and add this in routing file /other/consultations/:tab or other name, then you can get this key from params by this used name in this example:
const { tab } = useParams<{ tab?: string }>();

Reply:
@myslaf @daVitekPL this is resolved ?

Reply:
@qunabu I think so, this file is now removed and the logic moved to `src/pages/Consultations/index.tsx`